src/components/CohortBuilder/Summary/index.js

```diff
@@ -39,118 +39,95 @@ const CardSlot = styled(Card)`
-              tickInterval={4}
-              colors={[theme.chartColors.blue, theme.chartColors.purple]}
-              xTickTextLength={28}
-              legends={[
-                { title: 'Probands', color: theme.chartColors.blue },
-                { title: 'Family Members', color: theme.chartColors.purple },
-              ]}
-              padding={0.5}
-            />
-          </CardSlot>
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlot title="Most Frequent Diagnoses">
-            <HorizontalBar
-              style={{ maxWidth: '100px' }}
-              data={topDiagnosesBarMock}
-              indexBy="label"
-              keys={['probands', 'familyMembers']}
-              tooltipFormatter={mostFrequentDiagnosisTooltip}
-              sortByValue={true}
-              tickInterval={4}
-              colors={[theme.chartColors.blue, theme.chartColors.purple]}
-              xTickTextLength={28}
-              legends={[
-                { title: 'Probands', color: theme.chartColors.blue },
-                { title: 'Family Members', color: theme.chartColors.purple },
-              ]}
-              padding={0.5}
-            />
-          </CardSlot>
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlotPies>
-            <Pie
-              style={{ height: '42%', width: '50%', marginBottom: '10px', marginTop: '5px' }}
-              title={'Gender'}
-              data={demographicPiesMock.gender}
-              colors={[theme.chartColors.orange, '#FFFFFF']}
-            />
-            <Pie
-              style={{ height: '42%', width: '50%', marginBottom: '10px', marginTop: '5px' }}
-              title={'Ethnicity'}
-              data={demographicPiesMock.ethnicity}
-              colors={[theme.chartColors.darkblue, '#FFFFFF']}
-            />
-            <Pie
-              style={{ height: '42%', width: '50%' }}
-              title={'Race'}
-              data={demographicPiesMock.race}
-              colors={[theme.chartColors.lightpurple, '#FFFFFF']}
-            />
-            <Pie
-              style={{ height: '42%', width: '50%' }}
-              title={'Family Composition'}
-              data={demographicPiesMock.familyComposition}
-              colors={[theme.chartColors.lightblue, '#FFFFFF']}
-            />
-          </CardSlotPies>
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlot title="File Breakdown" />
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlot title="Age at Diagnosis" />
-        </Col>
-      </Row>
-    </Col>
-    <Col sm={12} md={3} lg={3}>
-      <LongCard title="Phenotypes">
-        <pre>{JSON.stringify(sqon, null, 2)}</pre>
-      </LongCard>
-    </Col>
-  </Row>
+const Summary = ({ theme, sqon, api }) => (
+  <QueriesResolver api={api} sqon={sqon} queries={[demographicQuery({ ...{ sqon } })]}>
+    {({ loading, data }) => {
+      return loading || !data ? (
```

Comment:
@ciaranschutte Does this mean that the user will have to wait for all queries to be resolved before seeing any results? Are we including error results as part of another ticket?

Reply:
@nyanofthemoon Yes. It means all queries will need to be resolved. I can include include error results in the global sqon ticket: https://app.zenhub.com/workspaces/oicr-sprints-59a846e9d6fb732a00631122/issues/kids-first/kf-portal-ui/1157